### PR TITLE
Regrid2#15

### PR DESCRIPTION
**Pull request by dnadeau4**
Issue #5 and #15 are in this  pull request.

### CODE REVIEW

**Comment by dnadeau4** [comment]
Fix #15

**Comment by doutriaux1** [comment]
@dnadeau4 opened or closed? It's not merged!

**Comment by dnadeau4** [comment]
I think my comment closed it by mistake.

**Comment by dnadeau4** [comment]
Please merge @doutriaux1

File: tests/test_gen_grid.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 import cdms2, numpy, os, sys
 import basetest
+import pdb
```
Comment (doutriaux1) [inline tests/test_gen_grid.py:9]: nope!

File: tests/test_regrid2.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 import cdms2, numpy.ma, regrid2 as regrid, os, sys
-from regrid2 import Regridder
+from regrid2 import Horizontal
```
Comment (doutriaux1) [inline tests/test_regrid2.py:3]: why did we change this? Shouldn't we have 2 tests?

**Review by doutriaux1 — CHANGES_REQUESTED** [review]

File: tests/test_regrid2.py (hunk 2 above)
Comment (dnadeau4) [reply tests/test_regrid2.py:3]: Because of this warning.
https://github.com/UV-CDAT/cdms/blob/master/regrid2/Lib/horizontal.py#L291-L294

**Pull request merged**